### PR TITLE
[CST-2626] Dev guidance for participants in frozen cohorts

### PR DESCRIPTION
**Pull request by tonyheadford**
### Context

- Ticket: [Jira](https://dfedigital.atlassian.net/browse/CST-2626)

### Changes proposed in this pull request
Added a doc based on notes from @ltello to help devs understand what is required to handle changes to participants in a cohort frozen for payments.

### Guidance to review
Please add any suggestions that would be helpful for devs dealing with these participants especially if dealing with support requests involving 2021 participants

### CODE REVIEW

**Comment by tonyheadford** [comment]
Closing because of build issue, will reopen

File: documentation/handling_participants_moving_from_a_frozen_cohort.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Where `cohort` is `Cohort.active_registration_cohort`
+
+## How do I move a 2021 participant to 2024?
+
+Once we are sure that a participant is eligible to be moved from 2021 to 2024 we use the `Induction::AmendParticipantCohort` service.  This performs lots of validations and safeguards that should not be skipped or performed manually.
+
+```ruby
+i = Induction::AmendParticipantCohort.new(participant_profile:,
+  																				source_cohort_start_year: 2021,
```
Comment (steventux) [inline documentation/handling_participants_moving_from_a_frozen_cohort.md:25]: The indentation here is out of bounds on the resulting MD doc.

**Review by steventux — APPROVED** [review]
Seems clear enough to me, easy to understand.
One indentation issue with a code snippet.

File: documentation/handling_participants_moving_from_a_frozen_cohort.md (hunk 1 above)
Comment (peteryates) [reply documentation/handling_participants_moving_from_a_frozen_cohort.md:25]: tabs-spaces

**Review by cwrw — APPROVED** [review]

File: documentation/handling_participants_moving_from_a_frozen_cohort.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Handling participants moving from a cohort frozen for payments to the active cohort
+
+To enable the 2021 cohort to be frozen for payments, participants that had not completed their induction could be moved to the 2024 cohort by following these steps in order:
+
+1. A change happens to the participant such as changing from `withdrawn` to `active` status, transferring to another school or a change of mentor, that indicates they are continuing with their training
```
Comment (ltello) [inline documentation/handling_participants_moving_from_a_frozen_cohort.md:5]: Actually, is the mentor who gets moved when they are associated an ECT out of the frozen cohort

File: documentation/handling_participants_moving_from_a_frozen_cohort.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+2. We verify that the participant is eligible to move
+3. We move the participant
```
Comment (ltello) [inline documentation/handling_participants_moving_from_a_frozen_cohort.md:7]: For school transfers, mentor-ECT associations, revalidations,...  the code will move the participant automatically to the active cohort from the frozen one if they are eligible.

**Review by ltello — APPROVED** [review]

**Pull request merged**